MerlinAU.sh

```diff
@@ -2599,13 +2623,16 @@ Please manually update to version $minimum_supported_version or higher to use th
             # Format release_version by removing the prefix '3004.' and the last '.0'
             formatted_release_version=$(echo $release_version | awk -F. '{print $2"."$3}')
 
+            # Define regex patterns for both versions
+            release_version_regex="[0-9]+\.$formatted_release_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"
+            current_version_regex="[0-9]+\.$formatted_current_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"
+
```

Comment:
This doesn't work with "**386.xx**" releases. The changelogs have the following format:

"**MAJOR**.**MINOR**_**PATCH** (DATE)"

For example:
**386.12_4** (21-Nov-2023)

Reply:
Hmm. Last one didn't work for 388.xx releases sadly.

Reply:
Is it possible to accommodate both standards via regex?

Something like: release_version_regex="([0-9]{3,4})\.([0-9]+)(_[0-9]+)? \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"

Reply:
Or here's an idea:

            release_version_regex="$formatted_release_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"
            current_version_regex="$formatted_current_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"

Drop the start of line requirement, and then just match by the date standard following with IS the same on both standards.

Reply:
> Or here's an idea:
> 
> ```
>         release_version_regex="$formatted_release_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"
>         current_version_regex="$formatted_current_version \([0-9]{1,2}-[A-Za-z]{3}-[0-9]{4}\)"
> ```
> 
> Drop the start of line requirement, and then just match by the date standard following which IS the same on both standards.
>
Yeah, I think that should work for both cases since it skips the 4-digit F/W codebase/branch number.